python.Dockerfile

```diff
@@ -1,32 +1,24 @@
-FROM        ubuntu:latest
+FROM        python:3.5-jessie
 MAINTAINER  James Turk <james@openstates.org>
 
 ARG DEBIAN_FRONTEND=noninteractive
-
 RUN apt-get update && apt-get install -y \
-    python3-dev \
-    python3-pip \
-    python3.5 \
-    git \
-    libpq-dev \
-    libgeos-dev \
-    libgdal-dev \
-    python-virtualenv
-
-RUN mkdir -p /opt/openstates.org/
+    build-essential \
+    libssl-dev \
+    binutils \
+    libproj-dev \
+    gdal-bin
```

Comment:
this is missing key libraries  included above

Reply:
Which ones exactly? GDAL was one of the dependency from what I could gather from the error messages but other than that I couldn't think of any